catala.opam

```diff
@@ -30,7 +30,6 @@ depends: [
   "cppo" {>= "1"}
   "dates_calc" {>= "0.0.7"}
   "dune" {>= "3.13"}
-  "js_of_ocaml-ppx" {= "4.1.0"}
```

Comment:
Unfortunately this is because of https://github.com/CatalaLang/catala/issues/442, we need to debug that